README.md

```diff
@@ -28,10 +28,19 @@ the documentation, which contains the unreleased features.
 ```jldoctest
 julia> using MuladdMacro
 
-julia> @macroexpand(@muladd k3 = f(t + c3*dt, @. uprev+dt*(a031*k1+a032*k2)))
+julia> @macroexpand(@muladd k3 = f(t + c3 * dt, @. uprev + dt * (a031 * k1 + a032 * k2)))
 :(k3 = f((muladd)(c3, dt, t), (muladd).(dt, (muladd).(a032, k2, (*).(a031, k1)), uprev)))
 
-julia> @macroexpand(@muladd integrator.EEst = integrator.opts.internalnorm((update - dt*(bhat1*k1 + bhat4*k4 + bhat5*k5 + bhat6*k6 + bhat7*k7 + bhat10*k10))./ @. (integrator.opts.abstol+max(abs(uprev),abs(u))*integrator.opts.reltol)))
+julia> @macroexpand(@muladd integrator.EEst = integrator.opts.internalnorm((update -
+                                                                            dt * (bhat1 * k1 +
```

Comment:
IMO this indentation is terrible. Is this really how it is supposed to look?

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/741 join the formatter discussions 😅. At this point, I think we need consistent formatting so I'm happy to suggest whatever the formatter throws out, but I do think "meh that could be better" for a few of the formatter rules and anyone who has a good idea for a new rule should just suggest it and we can get it in.